.github/workflows/build-v2.1.yml

```
@@ -199,6 +199,124 @@ jobs:
+          - [Sailfish v${VERSION}](https://www.nuget.org/packages/Sailfish/${VERSION})
+          - [Sailfish.TestAdapter v${VERSION}](https://www.nuget.org/packages/Sailfish.TestAdapter/${VERSION})
+          - [Sailfish.Analyzers v${VERSION}](https://www.nuget.org/packages/Sailfish.Analyzers/${VERSION})
+          EOF
+
+          echo "Manual release notes prepared:"
+          cat release_notes.md
+
+          # Clean up the manual notes file after use
+          rm RELEASE_NOTES.md
+
+      - name: Create Release
```

actionlint (1.7.7)

step must run script with "run" section or run action with "uses" section

(syntax-check)